amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-examples.adoc

```diff
@@ -1,21 +1,19 @@
 = Amazon S3 Connector 5.7 Examples
 :page-aliases: connectors::amazon/amazon-s3-connector-examples.adoc
```

Comment:
Please see the comment on the left. (I couldn't add it where I wanted to before.)